sponsorship_switzerland/models/account_invoice.py

```diff
@@ -93,3 +93,15 @@ def group_or_split_reconcile(self):
         """Reconcile given invoices with partner open payments.
         """
         return self._group_or_split_reconcile()
+
+    def action_invoice_open(self):
+        for invoice in self:
+            condition = [
+                ("transaction_id", "=", invoice.transaction_id),
+                ("state", "=", "open"),
```

Comment:
Maybe we want to include paid invoices here.
```suggestion
                ("state", "in", ["open", "paid"]),
```